edk2toolext/environment/extdeptypes/az_cli_universal_dependency.py

```diff
@@ -58,7 +58,7 @@ def VerifyToolDependencies(cls: "AzureCliUniversalDependency") -> None:
         if cls.VersionLogged:
             return
         results = StringIO()
-        RunCmd("az", "--version", outstream=results, raise_exception_on_nonzero=True)
+        RunCmd("az", "version", outstream=results, raise_exception_on_nonzero=True)
```

Comment:
With the format changing, simply parsing, line by line, the returned versions does not matter.

The reported version for azure-cli is now `'"2.54.0",'` instead of `2.54.0` (A set of double quotes and a comma exist when they should not). and for `azure-devops` the returned value is `"1.0.0"` instead of `1.0.0`. Can you look into fixing this?

I would suggest taking the input and running it through `data = json.loads(results)` and then looking for `azure-cli` and `azure-devops` in the created dictionary.

Reply:
@Nishanth1311 bump on this PR comment so we can get it updated and merged in.